Makefile

```diff
@@ -279,19 +291,27 @@ version: $(SASSC_BIN)
 test: test_build
 
 test_build: $(SASSC_BIN)
-	$(RUBY_BIN) $(SASS_SPEC_PATH)/sass-spec.rb -c $(SASSC_BIN) --impl libsass $(LOG_FLAGS) $(SASS_SPEC_PATH)/$(SASS_SPEC_SPEC_DIR)
+	$(RUBY_BIN) $(SASS_SPEC_PATH)/sass-spec.rb -c $(SASSC_BIN) --impl libsass \
+	--cmd-args "-I $(SASS_SPEC_PATH)/$(SASS_SPEC_SPEC_DIR)" \
```

Comment:
the "-I" stuff was fixed upstream now, but it doesn't hurt to duplicate it

Reply:
Seems I missed the sass-spec PR that fixed this!?

Reply:
Yeah, it was https://github.com/sass/sass-spec/pull/1438. I landed it after I got one thumbs-up since it was blocking other PRs

Reply:
Cool, glad it got fixed, but as you said, no harm in keeping the explicit option.